sharedbuffers/mapped_struct.py

```diff
@@ -141,7 +144,7 @@
     assert Py_GE == 5
 else:
     if six.PY3:
-        buffer = memoryview
+        buffer = make_memoryview
```

Comment:
`make_memoryview` is only used in py3

Reply:
You can't do that with make_memoryview as a cfunc. You have to make make_memoryview work in both versions, and have all the python2 conditionals inside it.

That's because you can't assign a cfunc to a python variable. I'm not sure how it's working for you now in fact. It should break hard.

Reply:
ping

Reply:
You may be able to avoid the `buffer` global and replace all uses of buffer with make_memoryview if you do that BTW. It will work better.